array_shift/README.md

```diff
@@ -0,0 +1,7 @@
+# Insert and shift middle index of array
```

Comment:
"Short summary or background information" portion of documentation isn't present